File: server/src/main/java/com/mio/server/models/WorldError.java
package com.mio.server.models;

public record WorldError(String value, int line, int column, ErrorType errorType, String description) {

    @Override
    public String toString() {
        return "WorldError{" +
                "value='" + value + '\'' +
                ", line=" + line +
                ", column=" + column +
                ", errorType=" + errorType +
                ", description='" + description + '\'' +
                '}';
    }
}

File: Boxworld/app/src/main/java/com/mio/cliente/boxworld/models/WorldError.kt
package com.mio.cliente.boxworld.models

import com.mio.cliente.boxworld.builders.ErrorBuilder
import java.io.Serializable

data class WorldError(val errorBuilder: ErrorBuilder): Serializable {

    var lexema = errorBuilder.lexema
    var line = errorBuilder.line
    var column = errorBuilder.column
    var type = errorBuilder.errorType
    var Description = errorBuilder.description

}
